@kiva/kv-components/vue/KvWwwHeader.vue

```diff
@@ -154,8 +153,9 @@ export default {
 		const searchOpen = ref(false);
 		const activeHeaderItem = ref(null);
 		const menuOpen = ref(false);
-		const menuComponent = ref(null);
```

Comment:
Why `shallowRef` here?

Reply:
+1 I can see shallowRef being useful for components references in general